index.html

```diff
@@ -79,6 +83,155 @@ <h2 class="headerTitle greenText">I’m a software developer</h2>
+            />
+            <h3 class="sectionCardTitle">Multi-Post Stories Gain+Glory</h3>
+            <div class="cardTags">
+              <p class="tagText">Ruby on rails</p>
+              <p class="tagText">CSS</p>
+              <p class="tagText">JavaScript</p>
+              <p class="tagText">HTML</p>
+            </div>
+            <button class="sectionBtn">See Project</button>
+          </div>
+          <div class="sectionCard">
+            <img
+              class="cardImage"
+              src="asset/image/icons/imgPlaceholder.png"
+              alt="project image placeholder"
+            />
+            <h3 class="sectionCardTitle">Multi-Post Stories Gain+Glory</h3>
+            <div class="cardTags">
+              <p class="tagText">Ruby on rails</p>
+              <p class="tagText">CSS</p>
+              <p class="tagText">JavaScript</p>
+              <p class="tagText">HTML</p>
+            </div>
+            <button class="sectionBtn">See Project</button>
+          </div>
+          <div class="sectionCard">
+            <img
+              class="cardImage"
+              src="asset/image/icons/imgPlaceholder.png"
+              alt="project image placeholder"
+            />
+            <h3 class="sectionCardTitle">Multi-Post Stories Gain+Glory</h3>
+            <div class="cardTags">
+              <p class="tagText">Ruby on rails</p>
+              <p class="tagText">CSS</p>
+              <p class="tagText">JavaScript</p>
+              <p class="tagText">HTML</p>
+            </div>
+            <button class="sectionBtn">See Project</button>
+          </div>
+          <div class="sectionCard">
+            <img
+              class="cardImage"
+              src="asset/image/icons/imgPlaceholder.png"
+              alt="project image placeholder"
+            />
+            <h3 class="sectionCardTitle">Multi-Post Stories Gain+Glory</h3>
+            <div class="cardTags">
+              <p class="tagText">Ruby on rails</p>
+              <p class="tagText">CSS</p>
+              <p class="tagText">JavaScript</p>
+              <p class="tagText">HTML</p>
+            </div>
+            <button class="sectionBtn">See Project</button>
+          </div>
+        </div>
+      </section>
+      <section class="innerSection" id="aboutSection">
+        <h3 class="sectionTitle">About Me</h3>
+        <p class="aboutMeText">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <button class="sectionBtn">Get My Resume</button>
+        <div class="sectionGrid">
+          <div class="sectionCard aboutCard">
+            <div class="iconHolder">
+              <img src="asset/image//icons/langauge.png" alt="langauge icons" />
+            </div>
+            <h3 class="sectionCardTitle">Languages</h3>
```

Comment:
Kindly add the technology categories ("Languages", "Frameworks" etc ...) in a `ul` tag each of them in a `li` tag , that will make your code more professional and also its better to use `ul` and `li` when listing elements✔😃